### PR TITLE
2485 remove the `/api/log` endpoint

### LINKED ISSUE #2485: Remove `/api/log` endpoint
# Description
The `/api/log` endpoint has been replaced. Remove the legacy endpoint since nothing uses it anymore.

# Tasks

- [ ] Remove the `/api/log` endpoint/resource. (0.25d)

### PR DESCRIPTION
**Pull request by falcon78921**
# What does this PR do?

Fixes #2485 

Became familiar with the `monkey` project recently. Thought I would contribute what I could, having gone through the setup/codebase.

This PR should remove the `/api/log` endpoint from Monkey Island. If there's something I'm missing, please let me know and I'll correct.

## PR Checklist
* [x] Have you added an explanation of what your changes do and why you'd like to include them?
* [ ] Is the TravisCI build passing?
* [x] Was the CHANGELOG.md updated to reflect the changes?
* [ ] Was the documentation framework updated to reflect the changes?
* [x] Have you checked that you haven't introduced any duplicate code?

## Testing Checklist

* [ ] Added relevant unit tests?
* [ ] Have you successfully tested your changes locally? Elaborate:
    > Tested by {Running the Monkey locally with relevant config/running Island/...}
* [ ] If applicable, add screenshots or log transcripts of the feature working

### CODE REVIEW

File: monkey/monkey_island/cc/resources/log.py [hunk 1]
```diff
@@ -1,35 +0,0 @@
-import json
-
-from bson import ObjectId
-from flask import request
-
-from monkey_island.cc.database import mongo
-from monkey_island.cc.resources.AbstractResource import AbstractResource
-from monkey_island.cc.resources.request_authentication import jwt_required
-from monkey_island.cc.services.log import LogService
```
Comment (cakekoa) [inline monkey/monkey_island/cc/resources/log.py:9]: This Log resource is actually the sole user of `LogService`, so we should remove `LogService` as well

**Review by cakekoa — COMMENTED** [review]
Hi @falcon78921, thank you for your contribution to Infection Monkey -- and welcome! The changes look good, but I added one comment. Might as well remove `LogService` if it's no longer used

File: monkey/monkey_island/cc/resources/log.py (hunk 1 above)
Comment (falcon78921) [reply monkey/monkey_island/cc/resources/log.py:9]: Thanks @cakekoa

**Comment by mssalvatore** [comment]
Merged in commit https://github.com/guardicore/monkey/commit/25eff5ee2478f001eb62673450abd1d6b80a7ea0

**Pull request closed without merging**